WorkloadManagementSystem/Client/ServerUtils.py

```diff
@@ -31,7 +31,7 @@ def getDBOrClient(DB, serverName):
 
 
 def getPilotAgentsDB():
-  serverName = 'WorkloadManagement/Pilots'
+  serverName = 'WorkloadManagement/PilotManager'
```

Comment:
This is really correcting a bug. Can you port this to rel-v6r22 please?

Reply:
I created a new PR for rel-v6r22